monitor/components.py

```diff
@@ -189,7 +188,7 @@ def nodes_table(nodes) -> (html.Table, List):
                                          'if': {
                                              'filter_query': f'{{Last Seen}} eq "{NOT_YET_CONNECTED_TO_NODE}"'
                                          },
-                                         'opacity': 0.45
+                                         # 'opacity': 0.45
```

Comment:
Any reason you don't want the opacity?

Reply:
Hmm, I may have meant to restore this line, but was trying to improve some visual artifacts occurring when a row is 100% opacity and also true black.